plugins/BEdita/Core/src/Model/Table/ObjectRelationsTable.php

```diff
@@ -0,0 +1,97 @@
+<?php
+namespace BEdita\Core\Model\Table;
+
+use Cake\ORM\Query;
+use Cake\ORM\RulesChecker;
+use Cake\ORM\Table;
+use Cake\Validation\Validator;
+
+/**
+ * ObjectRelations Model
+ *
+ * @property \Cake\ORM\Association\BelongsTo $Objects
+ * @property \Cake\ORM\Association\BelongsTo $Relations
+ * @property \Cake\ORM\Association\BelongsTo $Objects
+ *
+ * @method \BEdita\Core\Model\Entity\ObjectRelation get($primaryKey, $options = [])
+ * @method \BEdita\Core\Model\Entity\ObjectRelation newEntity($data = null, array $options = [])
+ * @method \BEdita\Core\Model\Entity\ObjectRelation[] newEntities(array $data, array $options = [])
+ * @method \BEdita\Core\Model\Entity\ObjectRelation|bool save(\Cake\Datasource\EntityInterface $entity, $options = [])
+ * @method \BEdita\Core\Model\Entity\ObjectRelation patchEntity(\Cake\Datasource\EntityInterface $entity, array $data, array $options = [])
+ * @method \BEdita\Core\Model\Entity\ObjectRelation[] patchEntities($entities, array $data, array $options = [])
+ * @method \BEdita\Core\Model\Entity\ObjectRelation findOrCreate($search, callable $callback = null, $options = [])
+ */
+class ObjectRelationsTable extends Table
+{
+
+    /**
+     * Initialize method
+     *
+     * @param array $config The configuration for the Table.
+     * @return void
+     */
+    public function initialize(array $config)
+    {
+        parent::initialize($config);
+
+        $this->table('object_relations');
+        $this->displayField('left_id');
+        $this->primaryKey(['left_id', 'relation_id', 'right_id']);
+
+        $this->belongsTo('Objects', [
```

Comment:
We should use different alias for `belongsTo` with objects table to avoid conflicts. Something as
```
$this->belongsTo('LeftObjects', [
    'foreignKey' => 'left_id',
    'joinType' => 'INNER',
    'className' => 'BEdita/Core.Objects'
]);

$this->belongsTo('RightObjects', [
    'foreignKey' => 'right_id',
    'joinType' => 'INNER',
    'className' => 'BEdita/Core.Objects'
]);